pkg/machine/qemu/config.go

```diff
@@ -42,6 +54,75 @@ type MachineVM struct {
 	UID int
 }
 
+type MachineVM struct {
+	// The command line representation of the qemu command
+	CmdLine []string
+	// HostUser contains info about host user
+	HostUser
+	// ImageConfig describes the bootable image
+	ImageConfig
+	// Mounts is the list of remote filesystems to mount
+	Mounts []Mount
+	// Name of VM
+	Name string
+	// PidFilePath is the where the PID file lives
+	PidFilePath MachineFile
+	// QMPMonitor is the qemu monitor object for sending commands
+	QMPMonitor Monitor
+	// ReadySocket tells host when vm is booted
+	ReadySocket MachineFile
+	// ResourceConfig is physical attrs of the VM
+	ResourceConfig
+	// SSHConfig for accessing the remote vm
+	SSHConfig
```

Comment:
Is it intentional that some structs are embedded? (e.g., SSHConfig or HostUser)

Reply:
Yeah